experimental/CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ All notable changes to experimental packages in this project will be documented
 
 * feat(otlp-transformer): Do not limit @opentelemetry/api upper range peerDependency [#4816](https://github.com/open-telemetry/opentelemetry-js/pull/4816) @mydea
 * feat(instrumentation-http): Allow to opt-out of instrumenting incoming/outgoing requests [#4643](https://github.com/open-telemetry/opentelemetry-js/pull/4643) @mydea
+* feat(instrumentation-http): Add support for client span semantic conventions 1.27 [#4940](https://github.com/open-telemetry/opentelemetry-js/pull/4940) @dyladan
```

Comment:
Oh, that should to go to unreleased :slightly_smiling_face:

Reply:
[e4ccec8](https://github.com/open-telemetry/opentelemetry-js/pull/4940/commits/e4ccec81b657128fd61cd389f9b3d58b8a115b8f)